drivers/clock_control/clock_control_nrf2_hsfll.c

```diff
@@ -192,6 +204,28 @@ static int api_get_rate_hsfll(const struct device *dev,
 
 static int hsfll_init(const struct device *dev)
 {
+	const struct hsfll_dev_config *dev_config = dev->config;
+	nrf_hsfll_trim_t trim = {
+		.vsup   = sys_read32(dev_config->trim_vsup),
+		.coarse = sys_read32(dev_config->trim_coarse),
+		.fine   = sys_read32(dev_config->trim_fine),
```

Comment:
Settint TCOEF trim is still missing. Are we aiming to fix it in this PR? If not we should create a ticket to make sure we don't forget.

Reply:
Created a ticket, this PR just moves the existing init code from SOC to the clock driver, so it arguably is out of scope :)